src/main/kotlin/com/example/Food_API/models/Food.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.Food_API.models
```

Comment:
You may not need this `Food.kt` data class, unless you're assuming to eventually convert the response body from the spoonacular API to this data type and return that to the consumer of your API.

Reply:
I removed the food class, and added the produces to get the getmapping, in a different branch and i'll be opening a PR for those changes soon.